devito/passes/clusters/cse.py

```diff
@@ -67,6 +67,15 @@ def _cse(maybe_exprs, make, mode='default'):
     # dependence involving `a`
     exclude = {i.source.access for i in scope.d_flow.independent()}
 
+    # Some expressions may be pre-processed nested reduction or Function-as-indices.
+    # that already created temporaries. For example
+    # Eq(u[x+1], u[f[x]+1]) would be preprocessed as
```

Comment:
by "preprocessed" you mean interpolation-handcrafted temporaries?

Reply:
nitpicking: use backtips to sorround inline code examples

that is

\`Eq(u[x+1], u[f[x]+1])\`